source/Sashimi.Tests/Sashimi.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="Octopus.Server.Tests.Extensibility" Version="12.1.1" />
-    <PackageReference Include="Sashimi.Tests.Shared" Version="10.0.0" />
```

Comment:
This version is being downgraded back to the 9.x line. The only [addition in 10.x](https://github.com/OctopusDeploy/Sashimi/compare/9.0.2...octopusdeploy:10.0.0) is an extra `feedId` parameter in the `ICreateTargetServiceMessageHandler` interface. But in the 2021.1 release, [the calling code](https://github.com/OctopusDeploy/OctopusDeploy/blob/release/2021.1/source/Octopus.Server/Orchestration/ServerTasks/Deploy/ActionDispatch/ServiceMessageHandlers/CreateDeploymentTargetOrchestrator.cs#L265) does not provide that parameter.